omero/developers/whatsnew.txt

```diff
@@ -1,27 +1,7 @@
-What's new for OMERO 5.1 for developers
+What's new for OMERO 5.2 for developers
 =======================================
 
-- The :doc:`/sysadmins/version-requirements` section provides extensive
```

Comment:
Isn't this initial point about the version requirements equally applicable to 5.2 and subsequent releases?  The following points were new 5.1 features and are fine to remove, but not sure about this one.